clients/src/main/resources/common/message/TxnOffsetCommitRequest.json

```diff
@@ -31,8 +33,15 @@
       "about": "The current producer ID in use by the transactional ID." },
     { "name": "ProducerEpoch", "type": "int16", "versions": "0+",
       "about": "The current epoch associated with the producer ID." },
+    { "name": "GenerationId", "type": "int32", "versions": "3+", "default": "-1", "ignorable": true,
+      "about": "The generation of the consumer." },
+    { "name": "MemberId", "type": "string", "versions": "3+", "ignorable": true,
+      "about": "The member ID assigned by the group coordinator." },
+    { "name": "GroupInstanceId", "type": "string", "versions": "3+", "ignorable": true,
```

Comment:
It's a bit inconsistent to set these as ignorable if the builder actually raises an exception. I would suggest making them non-ignorable and removing the checks inside the builder.